CrewChiefV4/PluginInstaller.cs

```diff
@@ -230,52 +257,49 @@ public void InstallOrUpdatePlugins(GameDefinition gameDefinition)
                     }
                 }
             }
-            //we have a gameInstallPath so we can go on with installation/updating assuming
+            //we have a gameInstallPath so we can go on with installation/updating assuming that the user wants to enable the plugin.
             if (Directory.Exists(gameInstallPath))
             {
                 if (gameDefinition.gameEnum == GameEnum.RF2_64BIT)
                 {
                     UserSettings.GetUserSettings().setProperty("rf2_install_path", gameInstallPath);
-                }
-                else if (gameDefinition.gameEnum == GameEnum.ASSETTO_32BIT || gameDefinition.gameEnum == GameEnum.ASSETTO_64BIT)
-                {
-                    UserSettings.GetUserSettings().setProperty("acs_install_path", gameInstallPath);
-                    string pythonConfigPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments), @"Assetto Corsa\cfg", @"python.ini");
                     try
                     {
-                        if (File.Exists(pythonConfigPath))
+                        string configPath = Path.Combine(gameInstallPath, @"UserData\player\CustomPluginVariables.JSON");
+                        if (File.Exists(configPath))
                         {
-                            bool found = false;
-                            string []lines = File.ReadAllLines(pythonConfigPath);
-                            for (int i = 0; i < lines.Length; i++ )
+                            string json = File.ReadAllText(configPath);
+                            Dictionary<string, Dictionary<string, int>> plugins = JsonConvert.DeserializeObject<Dictionary<string, Dictionary<string, int>>>(json);
+                            if (plugins.ContainsKey(rf2PluginFileName))
                             {
-                                if (lines[i].Equals("[CREWCHIEFEX]"))
-                                {
-                                    if (lines.Length >= i + 1)
-                                    {
-                                        if (lines[i + 1].Equals("ACTIVE=0"))
-                                        {
-                                            lines[i + 1] = "ACTIVE=1";
-                                        }
-                                        found = true;
-                                        break;
-                                    }
-                                }    
+                                //the whitespace is intended, this is how the game writes it.
+                                plugins[rf2PluginFileName][" Enabled"] = 1;
```

Comment:
I can add a message box asking if the user wants to auto enable the plugin incase its disabled or not present